catalyst/dl/callbacks/criterion.py

```diff
@@ -57,7 +92,79 @@ def on_batch_end(self, state: RunnerState):
             }
         )
 
-        self._add_loss_to_state(state, loss)
+        _add_loss_to_state(self.prefix, state, loss)
+
+
+class CriterionAggregatorCallback(Callback):
+    """
+    This callback allows you to aggregate the values of the loss
+    (by ``sum`` or ``mean``) and put the value back into ``state.loss``.
+    """
+    def __init__(
+        self,
+        prefix: str,
+        loss_keys: List[str] = None,
+        loss_aggregate_fn: str = "sum",
+        multiplier: float = 1.0
+    ) -> None:
+        """
+        Args:
+            prefix (str): new key for aggregated loss.
+            loss_keys (List[str]): If not empty, it aggregates
+                only the values from the loss by these keys.
+            loss_aggregate_fn (str): function for aggregation.
+                Must be either ``sum`` or ``mean``.
+            multiplier (float): scale factor for the aggregated loss.
+        """
+        super().__init__(CallbackOrder.Criterion + 1)
+        assert prefix is not None and isinstance(prefix, str), \
+            "prefix must be str"
+        self.prefix = prefix
+
+        if not isinstance(loss_keys, list) and loss_keys is not None:
+            loss_keys = [loss_keys]
+        self.loss_keys = loss_keys
+
+        self.multiplier = multiplier
+        if loss_aggregate_fn == "sum":
+            self.loss_fn = lambda x: torch.sum(torch.stack(x)) * multiplier
+        elif loss_aggregate_fn == "mean":
+            self.loss_fn = lambda x: torch.mean(torch.stack(x)) * multiplier
+        else:
+            raise ValueError("loss_aggregate_fn must be `sum` or `mean`")
+
+        self.loss_aggregate_name = loss_aggregate_fn
+
+    def _get_losses(self, loss: Any) -> List[torch.Tensor]:
```

Comment:
maybe,
```python
_process_loss
_preprocess_loss
_aggregate_loss
```

just because you already have loss in params, so.... there should not be any `get` :)